File: C programmes/_MACRO_/fibonacci.c
#include<stdio.h>
void fibonacci(int);
int main()
{
    int n;
    printf("enter number of elen=ments:\n");
    scanf("%d", &n);
    fibonacci(n);
    return 0;
}
void fibonacci(int num)
{
    int n1 = 0, n2 = 1, n3, count;
    printf("fibonacci series:\n");
    printf("%d\t%d\t", n1, n2);
    for (count = 3; count <= num;count++)
    {
        n3 = n1 + n2;
        printf("%d\t", n3);
        n1 = n2;
        n2 = n3;
    }
}

File: JAVA/Fibonacci.java
class Fibonacci{

	static void fib(int num){
		int n1=1;
		int n2=1;
		int n3;
		System.out.print("Fibonacci series : "+n1+" "+n2+" ");
		for(int i=3;i<=num;i++)
		{
			n3=n1+n2;
			System.out.print(n3+" ");
			n1=n2;
			n2=n3;
		}
	}

	public static void main(String [] args){
		int n=10;
		//Fibonacci f = new Fibonacci();
		fib(n);
	}
}